src/settings.go

```diff
@@ -14,6 +14,7 @@ type Settings struct {
 	ConsoleTitle         bool              `json:"console_title"`
 	ConsoleTitleStyle    ConsoleTitleStyle `json:"console_title_style"`
 	ConsoleTitleTemplate string            `json:"console_title_template"`
+	EnableHyperlink      bool              `json:"enable_hyperlink"`
```

Comment:
Is there a side-effect of having this enabled but not supported in your terminal?

Reply:
yes I think your prompt won't be happy if the terminal you use don't support hyperlink. I read various threads and it seems there's no way to know if the terminal supports it or now. So, by default it's disabled.

Reply:
If you check [here](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda), a majority of terminal already support it.